ack-controllers/instance/adoptedresources-crd.yaml

```diff
@@ -0,0 +1,232 @@
+# https://github.com/aws-controllers-k8s/s3-controller/blob/main/config/crd/common/bases/services.k8s.aws_adoptedresources.yaml
```

Comment:
What is the reason to fork this?

Reply:
That's a fair question... IDK 

updated to use remote.